docs/DeveloperGuide.md

```diff
@@ -301,11 +464,13 @@ Use case ends.
 
 > Windows, Linux, Unix, OS-X
 
-##### Private contact detail
-
-> A contact detail that is not meant to be shared with others
-
 ## Appendix E : Product Survey
```

Comment:
More efforts could have been placed in product evaluations - for wunderlist, Todo and Fantastical ~ although it might not seem like it but product evaluation is one of the most important component in software engineering ~ why? through other products' failures we can learn how to do things better and not make the same mistakes :) Most (not all) of the evals given are not exactly useful towards building your project right? so why not try to find strengths and weaknesses that you can learn from and apply to your project ~ If time permits, I do encourage you guys to take some time to re-eval and fill up the list, everyone having "cannot block task" isn't exactly helpful towards your project.